### PR TITLE
Improve options documentation for `babel-plugin-transform-runtime`

### PR DESCRIPTION
**Pull request by aaronang**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | no
| Major: Breaking Change?  | no
| Minor: New Feature?      | no
| Deprecations?            | no
| Spec Compliancy?         | no
| Tests Added/Pass?        | no
| Fixed Tickets            | 
| License                  | MIT
| Doc PR                   | yes
| Dependency Changes       | no

I screwed up in #5400. Here is the setup for how it should be. I still have to think about the descriptions though.

@existentialism

### CODE REVIEW

File: packages/babel-plugin-transform-runtime/README.md [hunk 1]
```diff
@@ -77,15 +77,53 @@ require("babel-core").transform("code", {
...
 
 The `runtime` transformer plugin does three things:
 
 * Automatically requires `babel-runtime/regenerator` when you use generators/async functions.
 * Automatically requires `babel-runtime/core-js` and maps ES6 static methods and built-ins.
-* Removes the inline babel helpers and uses the module `babel-runtime/helpers` instead.
+* Removes the inline Babel helpers and uses the module `babel-runtime/helpers` instead.
 
-What does this actually mean though? Basically, you can use built-ins such as `Promise`, `Set`, `Symbol` etc as well use all the Babel features that require a polyfill seamlessly, without global pollution, making it extremely suitable for libraries.
+What does this actually mean though? Basically, you can use built-ins such as `Promise`, `Set`, `Symbol`, etc. as well use all the Babel features that require a polyfill seamlessly, without global pollution, making it extremely suitable for libraries.
```
Comment (xtuc) [inline packages/babel-plugin-transform-runtime/README.md]: nit: [...] `Symbol`, etc. as well [...] -> [...] `Symbol`, etc, as well [...]

File: packages/babel-plugin-transform-runtime/README.md [hunk 2]
```diff
@@ -77,15 +77,53 @@ require("babel-core").transform("code", {
...
+
+`boolean`, defaults to `true`.
+
+Enables the generation of helpers in `moduleName`.
+
+### `polyfill`
+
+`boolean`, defaults to `true`.
+
+Transforms any `core-js` polyfill import with the runtime's polyfill.
```
Comment (existentialism) [inline packages/babel-plugin-transform-runtime/README.md]: Think we should expand on the aliasing aspect:

> Toggles whether or not new built-ins (`Promise`, `Set`, `Map`, etc.) are transformed to use a non-global polluting polyfill.
>
> For more information, see [core-js aliasing]().

File: packages/babel-plugin-transform-runtime/README.md [hunk 3]
```diff
@@ -77,15 +77,53 @@ require("babel-core").transform("code", {
...
+
+### `regenerator`
+
+`boolean`, defaults to `true`.
+
+Transforms any regenerator import with the runtime's regenerator.
```
Comment (existentialism) [inline packages/babel-plugin-transform-runtime/README.md]: Same as polyfill:

> Toggles whether or not generator functions are transformed to use a regenerator runtime that does not pollute the global scope.
>
> For more information, see [Regenerator aliasing]().

File: packages/babel-plugin-transform-runtime/README.md [hunk 4]
```diff
@@ -77,15 +77,53 @@ require("babel-core").transform("code", {
 });
 ```
 
+## Options
+
+### `helpers`
+
+`boolean`, defaults to `true`.
+
+Enables the generation of helpers in `moduleName`.
```
Comment (existentialism) [inline packages/babel-plugin-transform-runtime/README.md]: Maybe?

> Toggles whether or not inlined Babel helpers (`classCallCheck`, `extends`, etc.) are replaced with calls to [`moduleName`]().
>
> For more information, see [Helper aliasing]().

**Comment by hzoo** [comment]
good stuff 👍

**Pull request merged**